compute/hybrid/hybridcompute.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package hybridcompute
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	hybridnetwork "github.com/Azure-Samples/azure-sdk-for-go-samples/network/hybrid"
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/compute/mgmt/compute"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+const (
+	publisher   = "Canonical"
+	offer       = "UbuntuServer"
+	sku         = "16.04-LTS"
+	errorPrefix = "Cannot create VM, reason: %v"
+)
+
+// fakepubkey is used if a key isn't available at the specified path in CreateVM(...)
+var fakepubkey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC7laRyN4B3YZmVrDEZLZoIuUA72pQ0DpGuZBZWykCofIfCPrFZAJgFvonKGgKJl6FGKIunkZL9Us/mV4ZPkZhBlE7uX83AAf5i9Q8FmKpotzmaxN10/1mcnEE7pFvLoSkwqrQSkrrgSm8zaJ3g91giXSbtqvSIj/vk2f05stYmLfhAwNo3Oh27ugCakCoVeuCrZkvHMaJgcYrIGCuFo6q0Pfk9rsZyriIqEa9AtiUOtViInVYdby7y71wcbl0AbbCZsTSqnSoVxm2tRkOsXV6+8X4SnwcmZbao3H+zfO1GBhQOLxJ4NQbzAa8IJh810rYARNLptgmsd4cYXVOSosTX azureuser"
```

Comment:
In general, I get nervous about this kind of thing. How fake is this key? Does it truly not exist, or is it just an extremely unprivileged thing?

Reply:
Agree. I had used the pattern in compute\compute.go to create VM, thought. I removed this fake public key. Instead, in case public key is not there, I won't populate LinuxConfiguration.